### ISSUE TITLE
Twitch cannot download VODs 

### ISSUE THREAD

**Issue by jay11800** [futo-org/grayjay-android#883]
finishing a download using the twitch plugin returns this error. 
good app otherwise, its the best way to view twitch streams and if you can download vods it is the only place where that is even possible 

![Screenshot_20240308_092507_Grayjay](https://github.com/futo-org/grayjay-android/assets/5142933/cb8efb42-d00f-4e35-ba90-69fd970fdc85)

**Comment by JDog2pt0** [comment]
Yeah, mine only downloads audio. Ah well, until more devs jump on board you just gotta play the waiting game.

**Comment by beatgammit** [comment]
> Ah well, until more devs jump on board you just gotta play the waiting game.

I'd love to help, but unfortunately:

> We are currently not accepting contributions to the core.
